perun-core/src/main/java/cz/metacentrum/perun/core/entry/SearcherEntry.java

```diff
@@ -53,10 +53,8 @@ public SearcherEntry() {
 	@Override
 	public List<User> getUsers(PerunSession sess, Map<String, String> attributesWithSearchingValues) throws InternalErrorException, AttributeNotExistsException, PrivilegeException, WrongAttributeAssignmentException {
 		// Authorization
-		if (!AuthzResolver.isAuthorized(sess, Role.PERUNADMIN) &&
-				!AuthzResolver.isAuthorized(sess, Role.PERUNOBSERVER)) {
+		if (!AuthzResolver.authorizedInternal(sess, "getUsers_Map<String, String>_policy", Collections.emptyList()))
```

Comment:
I would replace ", " in the policy name for "_" like in the other cases.

Reply:
Fixed